x-pack/platform/plugins/shared/alerting/server/task_runner/transform_action_params.ts

```diff
@@ -39,6 +39,7 @@ export interface TransformActionParamsOptions {
   flapping: boolean;
   aadAlert?: AADAlert;
   consecutiveMatches?: number;
+  dashboards?: Array<{ id: string }>;
```

Comment:
Not sure what exactly the task_runner does, but I am wondering if adding dashboards here is actually needed?